src/modules/nft/src/senders.js

```diff
@@ -0,0 +1,121 @@
+export default class Sender {
+  constructor({ address, web3, tokens, contractAddresses = [] }) {
+    this.web3 = web3;
+    this.contractAddresses = contractAddresses;
+    this.tokens = tokens;
+    this.address = address;
+    this.isCryptoKitties = false;
+
+    if (
+      this.contractAddresses.includes(
+        '0x06012c8cf97bead5deae237070f9587f8e7a266d'
```

Comment:
dont hardcode :( can we select the first token from the list or something?

Reply:
This is cryptoKitties and uses a different signature than the ERC721 standard